src/pages/index.astro

```diff
@@ -54,13 +54,13 @@ deadlines = deadlines
         />
 
         <Title level={3} className="font-bold text-3xl mb-4 mt-12 lg:mt-auto">
-          EuroPython 2023 Photos
+          On-site Volunteer
         </Title>
 
         <ButtonLink
-          href="https://www.flickr.com/photos/europython/collections/72157722021442529/"
+          href="/volunteers"
         >
-          Highlights of EuroPython 2023
+          Join us in Making EuroPython 2024
```

Comment:
I'm not sure if "Making" would be the right word choice. Maybe "Running"?

Reply:
Hmm I thought "making" involves broader and  more active collaboration. "Running" to me sounds a bit more bureaucratic. Does "Creating" or "Shaping" sound better to you?

Reply:
You are right. How about "Join us in Making EuroPython 2024 Happen"?

Reply:
![image](https://github.com/EuroPython/website/assets/6319134/e4769430-8ffc-4e8a-94d4-2ee889e7f177)
Haha I tested it before as well but I didn't like that on a full HD setting, the happen breaks off to the next line. I know it is not a biggie, but it bugged me a little :smile:

Reply:
Some alternatives could be:
- Help us Make EuroPython 2024 a Success
- Help us Make EuroPython 2024 a Reality
- Help Us Make EuroPython 2024 Memorable
- Help Us Make EuroPython 2024 Phenomenal

But "Shaping" is good as well